glui.cpp

```diff
@@ -31,6 +31,60 @@
 *****************************************************************************/
 #include "glui_internal_control.h"
 
+//PASS-THROUGHS
+//These are here so glui.h does not have to #include glut.h.
+void GLUI_Master_Object::set_glutDisplayFunc(void (*f)(void))
+{
+	glutDisplayFunc(f);
+}
+void set_glutTimerFunc(unsigned int millis, void (*f)(int value), int value)
+{
+	glutTimerFunc(millis,f,value);
+}
+void set_glutOverlayDisplayFunc(void(*f)(void))
+{
+	glutOverlayDisplayFunc(f);
+}
+void set_glutSpaceballMotionFunc(Int3_CB f)
+{
+	glutSpaceballMotionFunc(f);
+}
+void set_glutSpaceballRotateFunc(Int3_CB f)
+{
+	glutSpaceballRotateFunc(f);
+}
+void set_glutSpaceballButtonFunc(Int2_CB f)
+{
+	glutSpaceballButtonFunc(f);
+}
+void set_glutTabletMotionFunc(Int2_CB f)
+{
+	glutTabletMotionFunc(f);
+}
+void set_glutTabletButtonFunc(Int4_CB f)
+{
+	glutTabletButtonFunc(f);
+}
+void set_glutWindowStatusFunc(Int1_CB f)
+{
+	glutWindowStatusFunc(f);
+} 
+void set_glutMenuStatusFunc(Int3_CB f)
+{
+	glutMenuStatusFunc(f);
+}
+void set_glutMenuStateFunc(Int1_CB f)
+{
+	glutMenuStateFunc(f);
+}
+void set_glutButtonBoxFunc(Int2_CB f)  
+{
+	glutButtonBoxFunc(f);
+}
+void set_glutDialsFunc(Int2_CB f) 
+{
+	glutDialsFunc(f);
+}    
 
```

Comment:
It's a rare blunder, but wouldn't present a problem for a patch. This red tape seems a deficiency to me.

Yourself:
"I'd point out that it's for the maintainer to make decisions about stability of master, not usually the contributors."